deployer/src/main/java/com/percussion/services/pkginfo/impl/PSPkgInfoService.java

```diff
@@ -202,9 +203,11 @@ public PSPkgInfo findPkgInfo(String name)
       List<PSPkgInfo> pkgInfoList = null;
       Session session = getSession();
 
-         Criteria criteria = session.createCriteria(PSPkgInfo.class);
-         criteria.add(Restrictions.eq("descriptorName", name).ignoreCase());
-         pkgInfoList = criteria.list();
+      CriteriaBuilder builder = getSession().getCriteriaBuilder();
+      CriteriaQuery<PSPkgInfo> criteria = builder.createQuery(PSPkgInfo.class);
+      Root<PSPkgInfo> critRoot = criteria.from(PSPkgInfo.class);
+      criteria.where(builder.equal(builder.lower(critRoot.get("descriptorName")), name.toLowerCase()));
+      pkgInfoList = entityManager.createQuery(criteria).getResultList();
 
```

Comment:
I wasnt sure if I should straight return the entity manager or put it in teh variable then return it due to the additional changes in the return statement